### PR TITLE
Added new env variables for additional customization

### PR DESCRIPTION
**Pull request by justetz**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -4,52 +4,53 @@ require('dotenv').config();
...
+        await transporter.sendMail({
+            from: `"${FROM_NAME}" <${FROM_EMAIL}>`,
+            replyTo: REPLY_TO_EMAIL,
             to: req.body.to,
             subject: req.body.subject,
             text: req.body.body,
             // html: '<b> Test </b>'
-        };
-        await transporter.sendMail(message);
-        res.send({success: true, msg: 'Email successfully sent!'});
```
Comment (lramos15) [inline server.js:42]: Changing this response object is going to mess up the website side of things as it parses this object

File: server.js [hunk 2]
```diff
@@ -4,52 +4,53 @@ require('dotenv').config();
...
+        res.status(200).send('Email successfully sent!');
     } catch (err) {
         console.error(err);
-        res.send({success: false, msg: err});
+        res.sendStatus(500);
     }
 });
 
-app.get('/', (req, res) => {
-    res.send('Hello from the mail server :)');
```
Comment (lramos15) [inline server.js:50]: I liked having an being able to this endpoint as a sort of hey I'm up

File: .env.example [hunk 3]
```diff
@@ -1 +1,5 @@
 WEBSITE_VERIFICATION_TOKEN=POST REQUEST VERIFICATION TOKEN HERE
+FROM_EMAIL=events@rpiambulance.com
+FROM_NAME=RPIA Events
+REPLY_TO_EMAIL=officers@rpiambulance.com
+HOMEPAGE=http://rpiambulance.com
```
Comment (lramos15) [inline .env.example]: Https?

**Review by lramos15 — CHANGES_REQUESTED** [review]

File: server.js (hunk 2 above)
Comment (justetz) [reply server.js:50]: I agree. If it redirects you to rpiambulance.com, it's up!

**Review by lramos15 — APPROVED** [review]

**Pull request merged**